proto/reaction.proto

```diff
@@ -287,6 +287,13 @@ message ReactionRole {
     // A product can be any species produced by the reaction, whether desired
     // or undesired.
     PRODUCT = 8;
+    // When there is one intended chemical equation:
+    // - Set `is_desired_product=True` to indicate a desired product.
+    // - Use BYPRODUCT to indicate a chemical species that is an expected result of the reaction but is not the product of interest.
```

Comment:
one of the tests is failing because this line is too long; clang-format should fix it (should be called by format.sh, make sure you pay attention to the warnings about whether stuff needs to be installed)